### ISSUE TITLE
no Delete or Remove "Job Profile" on Recruiter Dashboard

### ISSUE THREAD

**Issue by achille005** [achille005/JobApp#32]
Please provide Remove or Delete button on Recruiter Dashboard

Question:
- How are you handling many jobs profiles on dashboard?

**Comment by Astrea123** [comment]
Please Specify  where you want to show the delete button on Recruiter dashboard

**Comment by achille005** [comment]
Job Profile detail page should be good

**Comment by achille005** [comment]
yes, try them on **Recruiter** dashboard. ( The optional Preferences fields)
